### DISCUSSION TITLE
http-probe including file uploads

### DISCUSSION THREAD

**Discussion by DocKDE** [slimtoolkit/slim#607] [Q&A]
Hi!
I'd like to include custom requests in my http probe and would need to send a multipart/form-data request including a file upload. I couldn't find how to do this, is it currently possible?

**Comment by kcq** [comment]
Yes, it is possible. You need to use the 'body_file' field when you use the --http-probe-cmd-file flag to define custom http probe commands.

**Comment by DocKDE** [comment]
Maybe I misunderstand but I don't think this solves my problem. If I use the `body_file` field, I then have to supply the body that is to be used in the HTTP probe request in a separate file. This body, however, would need to contain form-data including a file for uploading. Is there a way to embed a file to be uploaded within the `body_file`?
